### DISCUSSION TITLE
Mi Band 5 and Mi Band 6

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#2003] [Q&A]
Hi.  Is Mi Band 5 and Mi Band 6 are now supported?

I've read https://github.com/NightscoutFoundation/xDrip/issues/1872 , but still can't determine whether it's supported. 

Thanks

**Comment by Navid200** [comment]
The individual who has the most experience in supporting Mi Band devices suggested that he would create an app that would interface with xDrip:
https://github.com/NightscoutFoundation/xDrip/pull/1659#issuecomment-979924823

We are waiting for him to do that or tell us if he needs help.

So, unfortunately, xDrip still does not support Mi Band 5 nor 6.

**Comment by GLoren1** [comment]
Thanks. I see that @bigdigital  has a "forked" version available while he is working on the app.  If someone installs his version, can they go back to the official Night Scout version without losing settings?

**Reply by Navid200** [reply]
If you need to uninstall xDrip, you can backup the database:
https://navid200.github.io/xDrip/docs/Backup-Database.html

After installing xDrip, you can restore the database.  
https://navid200.github.io/xDrip/docs/Restore-Database.html

There is always a risk in doing this.  Forgetting to backup the database, accidentally deleting it, or not being able to restore it are some of the possible reasons for disappointment.

Uninstalling xDrip should always be done as a last resort.

**Comment by GLoren1** [comment]
Thank you. Does the sensor need to be restarted after an uninstall/new install?

**Reply by tolot27** [reply]
No. If you restore the database, it will be recognized as running. Nevertheless, the physical sensor/transmitter keeps in state 'started'.
